text-ui-test/EXPECTED.TXT

```diff
@@ -1,3 +1,7 @@
+    _______________________________________________________________________________
+     java.io.IOException: I can't retrieve any data. Creating new file..
```

Comment:
Oh is this intentional?

Reply:
Yes, this is because text-ui-test does not create any file yet. So there will always be no file to retrieve any data from.

Reply:
Did you test whether loading existing file works as well?